p/scripts/main.js

```diff
@@ -960,6 +960,12 @@ function init_stream(stream) {
 			return false;
 		}
 
+		el = ev.target.closest('.item.share > a[data-type="clipboard"]');
+		if (el) {	//Clipboard
+			navigator.clipboard.writeText(el.href);
```

Comment:
Could you please test `if (navigator.clipboard) {}` to fail gracefully with e.g. IE11?

Reply:
done